hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetCache.java

```diff
@@ -184,6 +186,26 @@ public FsDatasetCache(FsDatasetImpl dataset) throws IOException {
     this.memCacheStats = cacheLoader.initialize(this.getDnConf());
   }
 
+  public void restoreCache(String bpid) throws IOException {
+    if(!cacheLoader.isTransientCache() && getDnConf().getPersistCacheEnabled()) {
```

Comment:
if( -> if (
code format issue, please also correct it for other pieces of code.

Reply:
if(cacheLoader.isTransientCache()) {
      return;
    }
    PmemVolumeManager.getInstance().createBlockPoolDir();